pkgs/tools/text/kdiff3/default.nix

```
@@ -23,6 +23,11 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ extra-cmake-modules kdoctools wrapQtAppsHook ];
 
+  postInstall = lib.optionalString stdenv.isDarwin ''
+    cd $out
+    ln -s Applications/KDE/kdiff3.app/Contents/MacOS bin
```

```suggestion
    ln -s $out/Applications/KDE/kdiff3.app/Contents/MacOS $out/bin
```

Or put a `()` so that `cd $out` runs in a subshell.

---

oops thanks